### DISCUSSION TITLE
3d printed photobooth

### DISCUSSION THREAD

**Discussion by mgineer85** [photobooth-app/photobooth-app#13] [Show and tell]
Some pictures here:
![IMG_20230225_183837](https://user-images.githubusercontent.com/24636867/221376781-28ff8f69-0dd2-4410-b3d8-fb0778a1ed53.jpg)
![IMG_20230225_183912](https://user-images.githubusercontent.com/24636867/221376799-f7e2fd4f-2859-4891-9d5c-3fc797e75bc5.jpg)

Source: https://github.com/mgrl/photobooth-app/issues/5#issuecomment-1445191158

3D CAD files:
https://github.com/mgrl/photobooth-3d

**Comment by laguinch** [comment]
Hy, i print the photobooth but i have a problem with the camera support it doesn’t match with my imx519 camera. What nuts did you take too fix the screen ? Thx

**Reply by mgineer85** [reply]
Screen is with m3 in m3 brass inserts. The camera module just with m2 screws.

**Reply by laguinch** [reply]
the M3 screw are to big and M2 are too small :(

**Reply by mgineer85** [reply]
That would be Strange. For camera you got that little holder printed?

**Reply by laguinch** [reply]
yes but it seems that my camera is different https://www.kubii.com/fr/cameras-capteurs/3793-camera-af-16mp-pour-raspberry-pi-3272496312722.html

**Reply by mgineer85** [reply]
Indeed I do not know this camera. The original pi cameras are supported and the imx519 by arducam. 
If you can make it work let me know which changes you made.

**Reply by Striker62** [reply]
can you offer the stl files for download ? I try to get all files out of the step file with fussion 360. but i did not get all parts out.

**Reply by mgineer85** [reply]
Please stop using old threads for different than the original question/topic.

The stl are not provided as slicer can use step nowadays. Load the step in prusaslicer and split the object to parts.
